.github/workflows/test.yml

```diff
@@ -93,3 +93,37 @@ jobs:
         echo
       env:
         COMBINATION: 1
+
+  install-dir-test:
+    runs-on: windows-latest
+    name: 'Check install directory behaviour'
+
+    strategy:
+      matrix:
+        include:
+          - install-dir: C:\tools\cygwin
+          - install-dir: D:\cygwin64
+
+    steps:
+    - run: git config --global core.autocrlf input
+      shell: pwsh -command ". '{0}'".
```

Comment:
Is this line needed? I don't see the default shell: being changed?

Reply:
No, you're quite right, I'd copied that without understanding why it was there originally. I'll correct and respin shortly...